docu/SICKAppSpaceCodingStarterKit_Documentation.md

```diff
@@ -169,9 +185,9 @@ If you want to use different modules in one single app, you could do it as follo
 
 4. Repeat this for all wanted modules
 
-# PRO / CONS of different approaches
+## PRO / CONS of different approaches
```

Comment:
Please add a "S" to PRO in all instances where you use this phrase.

Reply:
Done